lib/misc/cells.js

```diff
@@ -0,0 +1,64 @@
+(function() {
+  var scopes, values;
+
+  scopes = require('./scopes');
+
+  values = require('underscore-plus').values;
+
+  module.exports = {
```

Comment:
It's better to prefix functions you want exported with `export`, i.e.
`export function getRange(ed) {...}`.